pulseguardian/web.py

```diff
@@ -171,7 +173,7 @@ def index():
         if g.user.pulse_users:
             return redirect('/profile')
         return redirect('/register')
-    return render_template('index.html')
+    return render_template('index.html',config=config)
```

Comment:
Space after comma.

Also, there's a lot of sensitive information in `config`. Perhaps it's better to pass in specific variables to avoid (or at least reduce the risk of) accidentally exposing the wrong ones.